incubator/binding-grpc/src/main/java/io/aklivity/zilla/runtime/binding/grpc/internal/config/GrpcServiceConfig.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021-2022 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.grpc.internal.config;
+
+
+import java.util.Set;
+
+import org.agrona.collections.ObjectHashSet;
+
+public class GrpcServiceConfig
+{
+    public final Set<GrpcMethodConfig> methodConfigs;
+    public final String serviceName;
+
+    public GrpcServiceConfig(
+        String serviceName)
+    {
+        this.serviceName = serviceName;
+        this.methodConfigs = new ObjectHashSet<>();
+    }
+
+    public void setMethodConfig(
+        GrpcMethodConfig method)
+    {
+        methodConfigs.add(method);
+    }
```

Comment:
Usually the config types are fairly immutable. Any reason these `methods` (configs) cannot be passed to the constructor instead?